test/pytest/test_bridge.py

```diff
@@ -76,97 +94,156 @@ async def assert_msg(self, expected_channel, **kwargs) -> None:
         msg = await self.next_msg(expected_channel)
         assert msg == msg | {k.replace('_', '-'): v for k, v in kwargs.items()}
 
+    # D-Bus helpers
+    internal_bus: str = ''
+
+    async def open_internal_bus(self):
+        if not self.internal_bus:
+            self.internal_bus = await self.check_open('dbus-json3', bus='internal')
+        assert self.protocol.open_channels[self.internal_bus].bus == self.protocol.internal_bus.client
+        return self.internal_bus
+
+    def send_bus_call(self, bus: str, path: str, iface: str, name: str, args: list) -> str:
```

Comment:
Isn't the first argument the channel identifier?  `self.send_json(channel_id, ...)`.  It is called `bus` here.

Reply:
Yes.  It's the channel ID of the bus.  I could rename it.

Reply:
Ahh, it's the channel for the bus... got it.